edgelet/contrib/debian/control

```diff
@@ -2,8 +2,7 @@ Source: iotedge
 Section: admin
 Priority: extra
 Maintainer: Azure IoT Edge Devs
-Build-Depends: debhelper (>=9),
-               dh-systemd
+Build-Depends: debhelper (>=9.20160709)
```

Comment:
I know this is the minimum we need, but should we make it higher?  (debhelper version is 10.2.5 in Stretch)

Reply:
Good catch. Shall update.